### PR TITLE
Update README.md to be more correct

### PR DESCRIPTION
**Pull request by deroneriksson**
Change algorithm categories from 5 to 6 (add Survival Analysis).
Remove whitespace from end of lines.
Minor text updates.
Replace triple back ticks with single back ticks.
Update perc.csv info to make more sense.

### CODE REVIEW

**Comment by deroneriksson** [comment]
Rendered in GitHub markdown [here](https://gist.github.com/deroneriksson/b4db8c8358b008862751).

File: README.md [hunk 1]
```diff
@@ -49,8 +49,9 @@ SystemML computations can be executed in a variety of different modes. To begin
 can be operated in Standalone mode on a single machine, allowing data scientists to develop
 algorithms locally without need of a distributed cluster. In order to scale up, algorithms can also be distributed
 across Spark or Hadoop.
```
Comment (dusenberrymw) [inline README.md]: ...be distributed across a cluster using Spark or Hadoop...

File: README.md [hunk 2]
```diff
@@ -49,8 +49,9 @@ SystemML computations can be executed in a variety of different modes. To begin
...
-This flexibility allows the utilization of an organization's existing resources and expertise. In addition, SystemML
-can be operated via Scala, Java, and Python. SystemML also features an embedded API for scoring models.
+This flexibility allows the utilization of an organization's existing resources and expertise.
+In addition, SystemML features a Spark MLContext API that allows for programmatic interaction via Scala and Java.
```
Comment (dusenberrymw) [inline README.md]: Supposedly we also can interact with MLContext via Python using [SystemML.py](https://github.com/apache/incubator-systemml/blob/master/src/main/java/org/apache/sysml/api/python/SystemML.py).

**Comment by dusenberrymw** [comment]
"SystemML will build on Linux, MacOS, or Windows, and requires Maven 3 and Java 7 (or higher)." -> "SystemML will build on Linux, MacOS, or Windows, and requires Maven 3 and Java 7 (or higher; we prefer Java 8 for improved performance with multi-threaded reads)."

File: README.md [hunk 3]
```diff
@@ -98,20 +99,20 @@ If required, please install the following packages in R:
 ## Running SystemML in Standalone Mode
 
 SystemML can run in distributed mode as well as in local standalone mode. We'll operate in standalone mode in this
-guide. 
-After you built SystemML from source (```mvn clean package```) the standalone mode can be executed either on Mac/Unix 
-using the ```./bin/systemml``` script or on Windows using the ```.\bin\systemml.bat``` batch file. 
+guide.
+After you built SystemML from source (`mvn clean package`), the standalone mode can be executed either on Mac/Unix
```
Comment (dusenberrymw) [inline README.md]: "After you build SystemML from source..."

File: README.md [hunk 4]
```diff
@@ -98,20 +99,20 @@ If required, please install the following packages in R:
...
+using the `./bin/systemml` script or on Windows using the `.\bin\systemml.bat` batch file.
```
Comment (dusenberrymw) [inline README.md]: "...script, or on Windows..." (comma will help readability)

File: README.md [hunk 5]
```diff
@@ -324,21 +321,21 @@ to the value obtained from the model training phase.
...
-	
+
 	./bin/systemml ./scripts/algorithms/LinearRegDS.dml -nvargs X=linRegData.train.data.csv Y=linRegData.train.labels.csv B=betas.csv fmt=csv
-	
+
 	./bin/systemml ./scripts/algorithms/GLM-predict.dml -nvargs X=linRegData.test.data.csv Y=linRegData.test.labels.csv B=betas.csv fmt=csv
 
 
-In this example, we've seen a small part of the capabilities of SystemML. For more detailed information, please 
+In this example, we've seen a small part of the capabilities of SystemML. For more detailed information, please
```
Comment (dusenberrymw) [inline README.md]: Let's add a "conclusion header" here:

### Conclusion

In this example, we've seen a small part...

**Comment by dusenberrymw** [comment]
@deroneriksson Awesome changes.  I read through the whole page again, and added some additional changes that we might as well make at the same time.

**Comment by deroneriksson** [comment]
@dusenberrymw Thank you for reviewing. I made the requested changes. I did not mention Python at this stage since I haven't run SystemML.py before. Its status might be a good question for the dev mailing list. I'll go ahead and merge.

**Pull request closed without merging**